scripts/startResourceServer.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable consistent-return, no-console */
+
+const fs = require('fs');
+const { exec } = require('child_process');
+const path = require('path');
+
+function startResourceServer() {
+  const dir = path.join(__dirname, '../samples-java-spring-mvc');
```

Comment:
I believe its safer to break out the `..` to its own argument, `(__dirname, '..', 'samples-java-spring-mvc')`